examples/query-actions/query_actions.md

```diff
@@ -1,12 +1,12 @@
 # Query actions
 
-The clauses in a query action are executed in the same way as the clauses in a query expression.
+Query action allows us to use the functionalities of query expression to iteratively execute a code block like foreach statement. Query action starts with the from clause and ends with the do clause. It can contain all the intermediate clauses in query expression.
```

Comment:
```suggestion
A query action allows you to use the functionalities of a query expression to iteratively execute a code block like a `foreach` statement. A Qquery action starts with the `from` clause and ends with the `do` clause. It can contain all the intermediate clauses in a query expression.
```